lib/coherence/config.ex

```diff
@@ -143,7 +143,7 @@ defmodule Coherence.Config do
     case get_application_env :email_reply_to do
       nil ->
         case {get_application_env(:email_reply_to_name), get_application_env(:email_reply_to_email)} do
-          {nil, nil} -> nil
```

Comment:
In user_email.ex, the function `add_reply_to` expects a return of `nil`, not `{nil, nil}`, so I think the spec is wrong, not the implementation. If you change the PR, I'll merge. Thanks!!